packages/office-ui-fabric-react/src/components/ThemeGenerator/ThemeGeneratorPage.tsx

```diff
@@ -40,7 +40,7 @@ const BackgroundImageUriKey = 'backgroundImageUri';
 const BackgroundOverlayKey = 'backgroundOverlay';
 
 export class ThemeGeneratorPage extends React.Component<any, IThemeGeneratorPageState> {
```

Comment:
change this to extend BaseComponent, then if you're using setTimeout you can use this._async.setTimeout